src/main/java/com/example/demo/db/repo/ScheduleRepo.java

```diff
@@ -12,12 +13,14 @@ public interface ScheduleRepo extends CrudRepository<Schedule,Integer> {
      * @param week
      * @return List
      */
+    @Query("SELECT s FROM Schedule s WHERE s.weekNumber = ?1")
     List<Schedule> getSchedulesByWeekNumber(Integer week);
```

Comment:
This is not needed. The CrudRepository class auto create this queries 
```suggestion
            List<Schedule> getSchedulesByWeekNumber(Integer weekNumber);
```
```suggestion
    Collection<Schedule> getSchedulesByWeekNumber(Integer weekNumber);
```